sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/OptimizerLoggingSuite.scala

```diff
@@ -152,4 +153,19 @@ class OptimizerLoggingSuite extends PlanTest {
       verifyLog(Level.INFO, Seq("Batch Has No Effect"))
     }
   }
+
+  test("SPARK-50329: toString for InSet should be valid for unresolved plan") {
+    withSQLConf(
+      SQLConf.PLAN_CHANGE_LOG_LEVEL.key -> "INFO"
```

Comment:
why do we need to set this conf in the test?

Reply:
I can remove it, it is useless here.